modules/opengl/src/geometry/meshgl.cpp

```diff
@@ -81,6 +81,7 @@ const Mesh::MeshInfo& MeshGL::getMeshInfoForIndexBuffer(size_t idx) const {
               "MeshGL::getMeshInfoForIndexBuffer(): index out of bounds");
     return indexBuffers_[idx].first;
 }
+const Mesh::MeshInfo& MeshGL::getDefaultMeshInfo() const { return defaultMeshInfo_; }
```

Comment:
Why not 
```suggestion
const Mesh::MeshInfo& MeshGL::getDefaultMeshInfo() const { return owner->getDefaultMeshInfo(); }
```

Reply:
I wanted the representation to be slightly more self reliant. For other data you can for example create a Volume from a VolumeRAM. But we can't to that with meshes at the moment. And this does not fix that, but it's a step in that direction...